zipkin-common/src/main/scala/com/twitter/zipkin/storage/QueryRequest.scala

```diff
@@ -0,0 +1,22 @@
+package com.twitter.zipkin.storage
+
+import com.twitter.util.Time
+
+/**
+ * @param serviceName Mandatory [[com.twitter.zipkin.common.Endpoint.serviceName]]
+ * @param spanName When present, only include traces with this [[com.twitter.zipkin.common.Span.name]]
+ * @param annotations Include  traces whose [[com.twitter.zipkin.common.Span.annotations]] value is in this list.
+ *                    This is an OR condition against [[binaryAnnotations]]
+ * @param binaryAnnotations Include traces whose [[com.twitter.zipkin.common.Span.binaryAnnotations]] include a
+ *                          String whose key and value are an entry in this map.
```

Comment:
given that it's also OR within the collection, a Map would be limiting struct, a `Set[(String, String)]` would be better